static/js/summarytable.js

```diff
@@ -0,0 +1,91 @@
+document.addEventListener("DOMContentLoaded", () => {
+    const metricsPerPage = 28;
+    let currentPage = 1;
+    let metrics = [];
+    let sortedMetrics = [];
+
+    const fetchMetrics = async () => {
+        const response = await fetch('http://playground.siglens.com:5122/metrics-explorer/api/v1/metric_names', {
```

Comment:
this should not point to playground, it was there just for testing, follow how we do for other api calls